lib/github.rb

```diff
@@ -43,7 +43,7 @@ def self.send_request(verb, url, params={})
       request.content_type = 'application/json'
     end
 
-    http.request(request)
+    Retriable.retriable { http.request(request) }
```

Comment:
this is a retriable without context? what is the default behavior?

Reply:
https://github.com/kamui/retriable#defaults